src/internal/user_api/requests.rs

```diff
@@ -17,19 +17,20 @@ use crate::{
     },
 };
 
+use crate::internal::auth_v2::AuthV2Builder;
 #[derive(Serialize, Deserialize, PartialEq, Debug)]
-pub struct PrivateKey(#[serde(with = "Base64Standard")] pub Vec<u8>);
+pub struct EncryptedPrivateKey(#[serde(with = "Base64Standard")] pub Vec<u8>);
```

Comment:
Seems like this change might have been better in a separate branch / PR.

Reply:
Fair observation. I'm not planning on backing it out unless you feel very strongly about it, though.

Reply:
Naw. Just a thought for the future.